homeassistant/components/spotify/media_player.py

```diff
@@ -598,7 +598,7 @@ def library_payload():
                 {"name": item["name"], "type": item["type"], "uri": item["type"]}
             )
         )
-    return library_info
+    return BrowseMedia(**library_info)
```

Comment:
```suggestion
    return BrowseMedia(
        title="Media Library",
        media_content_id="library",
        media_content_type="library",
        can_play=False,
        can_expand=True,
        children=[item_payload({"name": n, "type": t, "uri": t}) for t, n in LIBRARY_MAP.items()],
    )
```

Reply:
The children are already set correctly in the library_info dict. It's just the parent that needs fixing.